package.json

```diff
@@ -12,7 +12,9 @@
     "test": "node tests/runner",
     "mobile_test": "mocha tests/e2e/e2e_workflow.spec.js",
     "test:inspect": "node --inspect --debug-brk tests/runner",
-    "lint": "eslint ."
+    "lint-js": "eslint .",
+    "lint-ts": "tslint \"**/*.ts\" -c tslint.json -e \"**/__path__/*.ts\" -e \"node_modules/**\" -e \"tmp/**\"",
+    "lint": "npm-run-all -c lint-js lint-ts"
```

Comment:
If you're running them sequentially, just do `npm run foo bar` and skip a dependency.